httpcore/__init__.py

```diff
@@ -39,4 +39,4 @@
     "WriteError",
     "CloseError",
 ]
-__version__ = "0.8.2"
+__version__ = "0.8.3"
```

Comment:
Probably should update the CHANGELOG as well.

Reply:
Blergh, accidentally pushed that to master, but yup.